src/pages/Splash.vue

```diff
@@ -175,12 +176,23 @@ export default class Splash extends mixins(SplashMixin) {
 
     async moveToNextScreen() {
         if (process.platform === 'linux') {
-            if (!await (GameDirectoryResolverProvider.instance as LinuxGameDirectoryResolver).isProtonGame(this.activeGame)) {
+            const isProton = await (GameDirectoryResolverProvider.instance as LinuxGameDirectoryResolver).isProtonGame(this.activeGame);
+
+            if (isProton instanceof R2Error) {
```

Comment:
I'm thinking that maybe we'd benefit by having errors that can be logged to Sentry but having things not present in modals.

This is one example of an error we'd probably not want to show to a user as that means that relevant Steam files have been modified, and that's not really something that the user can solve themselves.